spec/core/block/schemas/PositionEnum.yml

```diff
@@ -0,0 +1,9 @@
+type: integer
+format: int32
+enum:
```

Comment:
i hate that 1 and 2 is used instead of "left" and "right", it was something suggested from a previous dev when i joined, should we update this in the API now that no clients have implemented this feature yet?